modin/pandas/series.py

```diff
@@ -1345,7 +1345,8 @@ def truncate(self, before=None, after=None, axis=None, copy=True):
         )
 
     def unique(self):
-        return self._query_compiler.unique().to_numpy().squeeze()
+        # Ensure we don't return a scalar to match shape of Panda's output
+        return np.atleast_1d(self._query_compiler.unique().to_numpy().squeeze())
```

Comment:
```suggestion
        return self.__constructor__(
            query_compiler=self._query_compiler.unique()
        ).to_numpy()
```
I think that instead of converting the scalar result into an array, we should avoid situations of getting scalar at all. We can construct a Series from the resulted query compiler, and then `.to_numpy()` method will return us the array with proper shape.